app/src/main/java/org/sopt/daangnmarket_android/ui/view/write/WriteFragment.kt

```diff
@@ -62,6 +58,8 @@ class WriteFragment : Fragment() {
             if (binding.etTitle.text.isNullOrBlank() || binding.etPrice.text.isNullOrBlank() || binding.etContent.text.isNullOrBlank() || binding.viewmodel?.selectedImageList?.value == null) {
```

Comment:
이곳 조건문도 고칠 수 있습니다.
```suggestion
            if (writeViewModel.writeTitle.value?.length != 0 || writeViewModel.writePrice.value?.length != 0 || writeViewModel.writeContent.value?.length != 0 || writeViewModel.selectedImageList.value?.size != 0) {
```

사실 제 코드나 유리님이 짠 코드나 크게 다른 것은 없습니다. 차이점은 단지 View에서 값을 읽어오느냐, ViewModel에서 값을 읽어오느냐의 차이일 뿐입니다. ViewModel과 LiveData를 공부하는 입장이기 때문에 이런 방식을 제안해보는 것입니다. :-) MVVM이라는 아키텍처 자체가 View를 구성하는데 필요한 data를 ViewModel이 관리하도록 설계하는 방식이기 때문입니다.

Reply:
아하 ViewModel에서 값을 읽어오는 거군요! WriteViewModel에 값을 입력해두고 거기서 data를 관리하니 ViewModel에서 값을 가져오도록 바꿔보겠습니다 :)